zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -0,0 +1,139 @@
+
+
+DEFAULT_ZBSADM_BIN_PATH = "/usr/local/bin/zbsadm"
+DEFAULT_ZBS_BIN_PATH = "/usr/bin/zbs"
+DEFAULT_ZBS_CONF_PATH = "/etc/zbs/client.conf"
+DEFAULT_ZBS_USER_NAME = "zbs"
+DEFAULT_STRIPE_VOLUME_COUNT = 64
+DEFAULT_STRIPE_VOLUME_UINT = "64KiB"
+
+
+def deploy_client(client_ip, client_password):
+    return shell.call("%s client deploy --host %s -p %s --silent" % (DEFAULT_ZBSADM_BIN_PATH, client_ip, linux.shellquote(client_password)))
```

Ruff (0.8.2)

`shell` may be undefined, or defined from star imports

(F405)